admin_manual/release_notes.rst

```diff
@@ -0,0 +1,266 @@
+================================
+ownCloud |version| Release Notes
+================================
+
+Recommended Setup for Running ownCloud
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+ownCloud runs on multiple operating systems and HTTP servers. For best 
+performance, stability, support, and full functionality we recommend:
+
+* Red Hat Enterprise Linux 7
+* MySQL/MariaDB
+* PHP 5.4 +
+* Apache 2.4
+
+Supported Platforms
+^^^^^^^^^^^^^^^^^^^
+
+* Hypervisors: Hyper-V, VMware ESX, Xen, KVM
+* Server: Windows 2008R2; 2012, Linux (Debian 6, RHEL / Centos 6 and 7, Ubuntu 
```

Comment:
There seems to be some other mix-ups as well. For example Debian 7 is missing.

As per our recent discussion with @MTRichards the following list is supported:

Debian 7 (ships with php 5.4)
RHEL / Centos 6.5 and 7 (till end of production phase 1 - with php 5.4 and
5.5 only)
Ubuntu 12.04 LTS and 14.04 LTS (for 1 year after next LTS version release -
with PHP 5.4 or 5.5 only )
Ubuntu 14.10
SLES 11, 12 (for 1 year after the next production release)